packages/block-editor/src/components/colors-gradients/dropdown.js

```diff
@@ -82,23 +65,21 @@ const LabeledColorIndicator = ( { colorValue, label } ) => (
 // or as a `Button` if it isn't e.g. the controls are being rendered in
 // a `ToolsPanel`.
 const renderToggle = ( settings ) => ( { onToggle, isOpen } ) => {
-	const { __experimentalIsItemGroup, colorValue, label } = settings;
+	const { colorValue, label } = settings;
 
-	// Determine component, `Item` or `Button`, to wrap color indicator with.
-	const ToggleComponent = __experimentalIsItemGroup ? Item : Button;
-	const toggleClassName = __experimentalIsItemGroup
-		? 'block-editor-panel-color-gradient-settings__item'
-		: 'block-editor-panel-color-gradient-settings__dropdown';
 	const toggleProps = {
 		onClick: onToggle,
-		className: classnames( toggleClassName, { 'is-open': isOpen } ),
-		'aria-expanded': __experimentalIsItemGroup ? undefined : isOpen,
+		className: classnames(
+			'block-editor-tools-panel-color-gradient-settings__dropdown',
```

Comment:
Based on the removed code, shouldn't this class be `block-editor-panel-color-gradient-settings__dropdown`? 

```suggestion
			'block-editor-panel-color-gradient-settings__dropdown',
```

That class is also targeted by a couple of e2e tests e.g. `heading.test.js` & `keep-styles-on-block-transforms.test.js`